MathlibTest/linear_combination.lean

```diff
@@ -363,3 +450,13 @@ example (K : Type*) [Field K] [CharZero K] {x y z p q : K}
             59049 * q ^ 7 * p * x ^ 2) *
           h₂
   exact test_sorry
+
+/- When `linear_combination` is used to prove inequalities, its speed is very sensitive to how much
```

Comment:
Could you add the slow version as well, with an appropriate heartbeeat bound, so we can (1) be sure it doesn't get substantially slower and (2) have it handy if/when we try to optimize `linear_combination`?

Reply:
I don't follow, sorry -- what do you mean by "the slow version"?  Do you mean the version with the default normalization (`ring`) rather than skipped normalization?

Reply:
Sorry, ignore that. I read "on an implementation with "minimal" typeclasses everywhere" as "on an example," but after a second's thought it's clear you mean an alternate implementation of `linear_combination`.

Reply:
Ah, got it!